gobblin-service/src/test/java/org/apache/gobblin/service/monitoring/GaaSObservabilityProducerTest.java

```diff
@@ -58,6 +61,10 @@ public void testCreateGaaSObservabilityEventWithFullMetadata() throws Exception
         TroubleshooterUtils.getContextIdForJob(flowGroup, flowName, flowExecutionId, jobName),
         createTestIssue("issueSummary", "issueCode", IssueSeverity.INFO)
     );
+    List<DatasetTaskSummary> summaries = new ArrayList<>();
+    summaries.add(new DatasetTaskSummary("/testFolder", 100, 1000));
+    summaries.add(new DatasetTaskSummary("/testFolder2", 1000, 10000));
```

Comment:
nit - since you both write and then re-read for verification, rather than hard-coding the same constants in both places, I'd save aside each `DatasetTaskSummary` here to be able to access from it down below when it comes to verification